src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataSerializerContext.cs

```diff
@@ -110,6 +126,11 @@ public ODataSerializerContext(ResourceContext resource, SelectExpandClause selec
         /// </summary>
         public SelectExpandClause SelectExpandClause { get; set; }
 
+        /// <summary>
+        /// Gets of sets the relative path to the resouce being serialized
+        /// </summary>
+        public IEdmPathExpression PathToProperty { get; set; }
```

Comment:
>IEdmPathExpression [](start = 15, length = 18)

maybe to use a Stack<string> is a better choice?

Reply:
Used queue instead. 

---
In reply to: [300159016](https://github.com/OData/WebApi/pull/1738#discussion_r300159016) [](ancestors = 300159016)